Clients/Xamarin.Interactive.Client.Web/ClientApp/Workbook.ts

```diff
@@ -0,0 +1,190 @@
+//
+// Author:
+//   Bojan Rajkovic <bojan.rajkovic@microsoft.com>
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as matter from "gray-matter"
+import * as uuidv4 from "uuid/v4"
+import * as isUUID from "is-uuid"
+import { WorkbookTarget, WorkbookSession, PackageDescription } from "./WorkbookSession"
+import { Map, List } from "immutable"
+import { loadAsync, JSZipObject } from "jszip"
+
+const defaultUti: string = "com.xamarin.workbook"
+const manifestPlatformToWorkbookTargetMap: Map<string, string> = Map({
+    "ios": "ios-xamarinios",
+    "macnet45": "mac-xamarinmac-full",
+    "macmobile": "mac-xamarinmac-modern",
+    "android": "android-xamarinandroid",
+    "wpf": "wpf",
+    "console": "console",
+    "dotnetcore": "console-netcore"
+});
+
+function getWorkbookTargetIdFromPlatform(platform: string): string {
+    return manifestPlatformToWorkbookTargetMap.get(platform.toLocaleLowerCase())
```

Comment:
These are not locale-specific strings (they're effectively enums). Just use `toLowerCase`.

Reply:
👍 Looks like bad autocomplete from VS Code. Never even _seen_ that method used.